tests.py

```diff
@@ -33,6 +34,7 @@ def test_annots(self):
         self.assertEqual(len(self.annots), len(EXPECTED))
         for a, expected in zip(self.annots, EXPECTED):
             self.assertEqual((a.page.pageno, a.tagname, a.contents, a.gettext()), expected)
+        self.assertEqual(self.annots[0].created, datetime(2019, 1 , 19, 21, 29, 42, tzinfo=timezone(-timedelta(hours=8))))
```

Comment:
I was a bit lazy to copy-paste all the dates to EXPECTED, since the actual parsing is tested by `test_decode_datetime`, so figured one test just for 'interface' is enough. But if you insist I can add to EXPECTED for consistency :)